src/editors/data/services/cms/urls.ts

```diff
@@ -61,6 +61,10 @@ export const courseAssets = (({ studioEndpointUrl, learningContextId }) => (
   `${studioEndpointUrl}/assets/${learningContextId}/`
 )) satisfies UrlFunction;
 
+export const libraryAssets = (({ blockId, studioEndpointUrl }) => (
+  `${studioEndpointUrl}/api/libraries/v2/blocks/${blockId}/assets/`
+)) satisfies UrlFunction;
```

Comment:
It might make sense to use this existing function instead?

https://github.com/openedx/frontend-app-authoring/blob/e118eb5971f0d52854aa5e545b4bf987ec7adc8d/src/library-authoring/data/api.ts#L65-L68

Reply:
Yeah, I saw it and I think to use it but I was quite concern because from my understanding of the code there is a separation (isolation) of the data, I mean, each component has their own data folder, for example:

https://github.com/openedx/frontend-app-authoring/blob/e118eb5971f0d52854aa5e545b4bf987ec7adc8d/src/library-authoring/data/api.ts#L54

and 

https://github.com/openedx/frontend-app-authoring/blob/e118eb5971f0d52854aa5e545b4bf987ec7adc8d/src/editors/data/services/cms/urls.ts#L43

I'm happy to make the change to avoid duplication

Reply:
I don't have a strong opinion on this. But the `editors/` code used to be in a separate repository, so it wasn't necessarily designed for each component to have its own data folder. I personally think it makes sense for all the library APIs to be in `src/library-authoring/data` regardless of where they're used.